packages/is-element-visible/index.ts

```diff
@@ -0,0 +1,33 @@
+import { FormTypes } from '@oneblink/types'
+import {
+  conditionalLogicService,
+  formElementsService,
+} from '@oneblink/sdk-core'
+
+export default function isShowing(
```

Comment:
is it worth making a recursive version now or wait until we need one?

Reply:
like to see if the parent is visible and so on?

Reply:
nevermind, `formElementsService.findFormElement` does the recursion.